languagetool-language-modules/de/src/main/resources/org/languagetool/resource/de/added.txt

```diff
@@ -13934,3 +13934,7 @@ Surfboards	Surfboard	SUB:NOM:PLU:NEU
 Surfboards	Surfboard	SUB:GEN:PLU:NEU
 Surfboards	Surfboard	SUB:DAT:PLU:NEU
 Surfboards	Surfboard	SUB:AKK:PLU:NEU
+Skorts	Skorts	SUB:NOM:PLU:NOG
```

Comment:
Laut Wikipedia gibt es ja den Singular "Skort", dann müsste in der zweiten Spalte "Skort" stehen.

Reply:
Gut Singular kann ich einfügen. Nehme ich die Annahme richtig an, dass das Wort weiblich ist?`Habe ich bisher nur die Pluralform gesehen und Wikipidia nutzt "ein".

Reply:
Ich denke, wir sollten solche Wörter nur einfügen, wenn wir eine gute Quelle haben. Wenn das in keinem Wörterbuch drinsteht, und wir das Geschlecht raten müssen, ist das ja schon ein Hinweis, dass es zu speziell ist.

Reply:
Bis auf Wikipedia nutzen alle Shops die Pluralform und da es eine Form aus Shorts und Hotpants ist, ging ich davon aus, dass es ein reines Pluralwort ist.

lass den Pull Request offen. Ich schaue mich mal, ob ich mehrere Treffer finde, die ein Geschlecht vermerken.